.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: build-docs
+on:
+  pull_request:
+    paths:
+      - .github/workflows/**
+      - docs/**
+  push:
+concurrency:
+  # Cancel other jobs in the same branch. We don't care whether CI passes
+  # on old commits.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+jobs:
+  build-docs:
+    if: github.event_name == 'pull_request'
+    name: build docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.8.x"
+          architecture: "x64"
+          cache: "pip"
+          cache-dependency-path: '**/requirements-doc.txt'
+      - run: pip install -r docs/requirements-doc.txt
+      - run: cd docs && sphinx-build -T -E -W -b html . build
+env:
+  MODIN_GITHUB_CI: true
```

Comment:
Not sure if this is needed.
```suggestion
```

Reply:
This line is also present in the `check-pr-title` 
https://github.com/modin-project/modin/blob/ff53445e1baa916d4b4ac6f2ecab21b86f5ccf02/.github/workflows/ci-required.yml#L17-L18

IMO this is redundant, however just wanted to be consistent with other yamls

Reply:
ok